src/components/PrintDimension.tsx

```diff
@@ -0,0 +1,55 @@
+import { Slider, Switch } from "antd";
+import React, { useEffect, useRef, useState } from "react";
+import "../styles/PrintDimension.css";
+import charts from "../dummyData/charts";
+
+import { API_DOMAIN } from "../config";
+
+interface PrintDimensionProps {
+  dimensionVallue: number;
+}
+const chartID = "";
+
+const PrintDimension: React.FC<React.PropsWithChildren<PrintDimensionProps>> = (
+  props
+) => {
+  useEffect(() => {
+    fetch(`${API_DOMAIN}dimensions/forchart/${chartID}`, {
+      method: "GET",
+      headers: {
+        Accept: "application/json",
+        Authorization: "",
+      },
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        //Get data and set it to Dimensions
+      })
+      .catch((e) => console.log(e));
+  }, []);
+
+  const allDimensions = charts[0].dimensions;
+  const something = allDimensions[props.dimensionVallue];
```

Comment:
Probably need a better variable name than "something" haha. Also, value is spelt with a double L, probably best to fix this.

Reply:
Ahaha forgot to change that